Week_6/homework/frontend/src/pages/HomePage.js

```diff
@@ -0,0 +1,140 @@
+import { useEffect, useState, Fragment } from "react";
+import { FaPlus } from "react-icons/fa";
+
+import {
+  AppShell,
+  Navbar,
+  Header,
+  Footer,
+  Aside,
+  Text,
+  MediaQuery,
+  Burger,
+  createStyles,
+  useMantineTheme,
+  Group,
+  Center,
+  Stack,
+  Input,
+  Button,
+  Checkbox,
+  Title
+} from "@mantine/core";
+
+export default function HomePage() {
+  const [tasks, setTasks] = useState([]);
+
+  useEffect(() => {
+
+    fetch("http://localhost:4000/", {
+      method: "GET", // *GET, POST, PUT, DELETE, etc.
+      mode: "cors", // no-cors, *cors, same-origin
+      cache: "no-cache", // *default, no-cache, reload, force-cache, only-if-cached
+      credentials: "same-origin", // include, *same-origin, omit
+      headers: {
+        "Content-Type": "application/json",
+        //Authorization: "Bearer " + "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VybmFtZSI6ImFiYzEyMyIsImVtYWlsIjoiYWJjMTIzQGdtYWlsLmNvbSJ9.kVAp_XhgpFH3Iyl9cnRGUjRFYiBGuRuyYAztbNcRVLs"
+      },
+      redirect: "follow", // manual, *follow, error
+      referrerPolicy: "no-referrer", // no-referrer, *no-referrer-when-downgrade, origin, origin-when-cross-origin, same-origin, strict-origin, strict-origin-when-cross-origin, unsafe-url
+    })
+    .then(resp => resp.json())
+    .then(json => setTasks(json))
+    .catch((err) => {
+        console.log(err);
+      });
+
+    //  console.log(window.localStorage.getItem("username"))
+
+  });
```

Comment:
When you use useEffect, please note the dependency lists you are using. You call this API multiple times unintentionally because you don't use a dependency list in React.

https://devtrium.com/posts/dependency-arrays